channels/rdpgfx/client/rdpgfx_main.c

```diff
@@ -625,24 +652,23 @@ static UINT rdpgfx_recv_end_frame_pdu(RDPGFX_CHANNEL_CALLBACK* callback,
 	gfx->TotalDecodedFrames++;
 	ack.frameId = pdu.frameId;
 	ack.totalFramesDecoded = gfx->TotalDecodedFrames;
-	IFCALLRET(context->PreFrameAck, sendAck, context, &ack);
```

Comment:
Why did you remove this callback?
Don't remember who it was but the idea was to control the callback sending by external parameters

Reply:
It was mine actually, I initially didn't want to expose FrameAck sending and some flag to control automatic frame acking, but when coming to implement it in the proxy, alongside needing to manually control capability sending, our current solution made more sense and was a cleaner, simpler implementation.
As the PreFrameAck was my idea in the first place, I was confident enough to know it can be removed.